common/api/appui-abstract.api.md

```diff
@@ -1224,9 +1224,9 @@ export enum StandardTypeNames {
     // (undocumented)
     Bool = "bool",
     // (undocumented)
-    Boolean = "boolean",
+    Boolean = "boolean",// locale specific
```

Comment:
What are these "locale specific" comments about?

Reply:
I guess just changes to api-extractor.

Reply:
yep